src/pages/react/overview.md

```diff
@@ -27,4 +27,10 @@ Coming soon.
 
 ## React Tooling
 
-Coming soon.
\ No newline at end of file
+Coming soon.
+
+## Native Tooling
+
+[Capacitor](https://capacitor.ionicframework.com) is the official cross-platform app runtime used to make your `Ionic React` web app run natively on iOS, Android, Electron, and the web. 
+
+While there are no known technical limitations to using `Ionic React` with [Cordova](https://cordova.apache.org/) plugins, Capacitor is officially recommended. **Note:** There are no plans to support `Ionic React` in the [Ionic CLI tooling](/docs/cli) at this time. For more details, please [see here](https://capacitor.ionicframework.com/docs/cordova).
```

Comment:
Should be something like
There are no plans to support Cordova integration for `Ionic React` in the [Ionic CLI tooling](/docs/cli) at this time.
The CLI will support Ionic React, but you can only enable Capacitor integration, not Cordova